vcstools/beam_calc.py

```diff
@@ -629,10 +627,11 @@ def find_sources_in_obs(obsid_list, names_ra_dec,
         for source_name in np.array(names_ra_dec)[:,0]:
             source_data = []
             for on, obsid in enumerate(obsid_list):
-                if source_name in beam_coverage[obsid].keys:
+                if source_name in beam_coverage[obsid].keys():
                     # Source was in the beam so include it
                     _, _, _, duration, _, centre_freq, channels = common_metadata_list[on]
                     enter_beam_norm, exit_beam_norm, max_power = beam_coverage[obsid][source_name]
+                    bandwidth = 1.28 * (channels[-1] - channels[0] + 1) # MHz
```

Comment:
I'm not sure how this is used later on, but is this robust for picket fence obs?

Reply:
Almost certainly not, although picket fence is definitely a more complicated beast, so I think you would have to treat each continuous channel range as its own "observation".

Reply:
As this is probably something larger than just this single piece of code, I'm going to suggest we ignore it here and possibly add it as an issue/to-do item in the future. Okay?

Reply:
I'm going to make that decision and say it's okay for this release.